src/test/java/fiori/component/TableCell.java

```diff
@@ -23,4 +23,31 @@ protected TableCell(WebElement element) {
     public Hyperlink getHyperlink() {
         return functionPassed(() -> new Hyperlink(getElement().findElement(By.className("th-tx-value"))));
     }
+
+    /**
+     * The method gets the hyperlink by checkbox from the table cell.
+     *
+     * @return Hyperlink
+     */
+    public Hyperlink getHyperlinkByCheckbox() {
```

Comment:
What does it mean getHyperlinkByCheckbox(). Please, change method name to getHyperlink()